src/main/java/com/softserveinc/Task01.java

```diff
@@ -16,9 +16,78 @@
  */
 public class Task01 {
     public static void main(String[] args) {
-                              //  I  | II | III | IV |  V  |  VI | VII | VIII | IX  |  X  | XI |  XII
-        double[] temperature = { -1.7, -1.0, 2.6,  9.0, 15.1, 19.4, 21.4, 21.2, 17.1, 11.1, 5.9,  1.4 };
+        //  I  | II | III | IV |  V  |  VI | VII | VIII | IX  |  X  | XI |  XII
+        double[] temperature = {-1.7, -1.0, 2.6, 9.0, 15.1, 19.4, 21.4, 21.2, 17.1, 11.1, 5.9, 1.4};
         // TODO: Write your code here
 
+        double minTemperature = temperature[0];
+        int indexMin = 1;
+        double maxTemperature = temperature[0];
+        int indexMax = 1;
+        double sumYearTemperature = 0;
+
+        for (int i = 1; i < temperature.length; i++) {
+
+            sumYearTemperature += temperature[i];
+            if (minTemperature > temperature[i]) {
+                minTemperature = temperature[i];
+                indexMin = i + 1;
+            }
+            if (maxTemperature < temperature[i]) {
+                maxTemperature = temperature[i];
+                indexMax = i + 1;
+            }
+        }
+
+        double avgTemperature = sumYearTemperature / 12;
```

Comment:
It is convenient to use the field length of the array instead of using the literal.
I mean `sumYearTemperature/temperature.length`